packages/xo-remote-parser/src/index.js

```diff
@@ -15,6 +18,17 @@ const parseOptionList = (optionList = '') => {
     optionList = optionList.substring(1)
   }
   const parsed = queryString.parse(optionList)
+
+  // bugfix for persisting error notification
+  if ('error' in parsed) {
```

Comment:
Any way to track where these values are incorrectly added to the remote?

Reply:
The logs we added show up when this value is added to the remote, but I was not able to reproduce the situation where it happens.